CMakeLists.txt

```
@@ -26,6 +26,9 @@ include(glibc)
 
 if (NOT CROSS_COMPILE_EDGESEC)
   CHECK_GLIBC_VERSION()
+  IF (GLIBC_VERSION STREQUAL "2.2.1")
+    MESSAGE(FATAL_ERROR "Unsupported version of glibc: ${GLIBC_VERSION}")
+  ENDIF ()
```

Would this be a better error message? (I'm guessing we want older versions to also fail)

Also, is it worth adding a comment why glibc <= 2.2.1 doesn't work?

```suggestion
  IF (GLIBC_VERSION VERSION_LESS_EQUAL "2.2.1")
    MESSAGE(FATAL_ERROR "Unsupported version of glibc: ${GLIBC_VERSION}")
  ENDIF ()
```

---

That make sense.

---

Huh, it's weird you encountered this issue. [glibc version 2.2.1 is apparently from 2001](https://sourceware.org/legacy-ml/libc-alpha/2001-01/msg00163.html), which is before OpenWRT, Ubuntu, or git even existed.